README.md

```diff
@@ -71,6 +71,19 @@ There are some known areas where we'd like to improve:
 
 Bounded numbers aren't very well handled. Consider, for example, the schema:
```

Comment:
If you think it's unclear, please just change this to

```suggestion
Bounded numbers (both integer and floating-point) aren't very well handled. Consider, for example, the schema:
```

I don't think the other changes add to clarity.

Reply:
floats build types, with degraded functionality.

ints cause typify to crash.

They are wildly different behaviours that, IMO, warrant separate explanation.

If you disagree, close this PR as the entire point of this PR is to help users know that they can & should use bounded float instead of bounded int when designing their APIs, if they want to use `typify` on their APIs, and are ok with it gracefully downgrading to not enforcing the min/max/etc.